experiments/testing_archive/test_tx_antenna_pattern_not_numpy.py

```diff
@@ -0,0 +1,66 @@
+#!/usr/bin/python
+
+# write an experiment that creates a new control program.
+
+import sys
+import os
+import numpy as np
+from scipy.constants import speed_of_light
+
+BOREALISPATH = os.environ['BOREALISPATH']
+sys.path.append(BOREALISPATH)
+
+import experiments.superdarn_common_fields as scf
+from experiment_prototype.experiment_prototype import ExperimentPrototype
+
+
```

Comment:
```suggestion
# pattern returned by tx_antenna_pattern is not a numpy array, but a list
# this will fail in check_slice() of ExperimentPrototype
```